pom.xml

```diff
@@ -115,6 +115,12 @@
       <version>1.5.3</version>
       <optional>true</optional>
     </dependency>
+    <dependency>
+      <groupId>org.jenkins-ci.plugins</groupId>
+      <artifactId>script-security</artifactId>
+      <version>1.16</version>
+      <optional>true</optional>
```

Comment:
Maybe it's better to make it a required dependency to avoid the classloading hell (see my comment regarding the Extension annotation)